src/index.js

```diff
@@ -0,0 +1,88 @@
+/** @format */
+
+import './style.css';
+import store from './modules/storage.js';
+import {
+  addTask, displayTask, clearField, input,
+} from './modules/utility.js';
+
+const addBtn = document.querySelector('#addBtn');
+const clearBtn = document.querySelector('#clearBtn');
+
+// Display
+window.addEventListener('load', () => {
+  displayTask(store.tasks);
+});
+
+// add
+addBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  if (input.value === '') {
+    // console.log("All Fields are required");
+  } else {
+    addTask(store.tasks);
+    displayTask(store.tasks);
+    clearField();
+  }
+});
+
+// remove
+document.addEventListener('click', (e) => {
+  if (e.target.classList.contains('trashBtn')) {
+    const removeTask = (index) => {
+      store.tasks = store.tasks.filter(
+        (task) => Number(task.index) !== Number(index),
+      );
+      localStorage.setItem('tasks', JSON.stringify(store.tasks));
+    };
+
+    removeTask(e.target.id);
+    displayTask(store.tasks);
+  }
+});
+
+// edit
+document.addEventListener('click', (e) => {
```

Comment:
Use long and descriptive names for the functions instead of calling them documents. The function should say what it does, as stated by: https://devinduct.com/blogpost/22/javascript-clean-code-best-practices.